modules/apps/29-fee/keeper/grpc_query_test.go

```diff
@@ -210,19 +227,40 @@ func (suite *KeeperTestSuite) TestQueryIncentivizedPacketsForChannel() {
 		{
 			"no packets for specified channel",
 			func() {
+				setEmptyChannel()
```

Comment:
also super small nit here, but since the `setEmptyChannel`function is only used once it might be more readable/explicit to just have the `setEmptyChannel` logic here in its entirety.

Reply:
Ah thanks for catching it! It was used twice but then one of the tests got refactored and I forgot to remove this :)